test/include/seqan3/test/performance/units.hpp

```diff
@@ -12,14 +12,16 @@
 
 #pragma once
 
+#include <seqan3/std/ranges>
+
```

Comment:
there already is a std/ranges include below the benchmark include? I think you can delete this one again